test/verify/check-users

```diff
@@ -858,6 +858,75 @@ class TestAccounts(MachineCase):
         b.wait_not_present("#groups-create-dialog")
         b.wait_visible('#groups-list td[data-label="Group name"]:contains("saturn")')
 
+        # Deleting group will go back to overview
+        b.go("/users#/group/saturn")
+        b.wait_val("#group-name", "saturn")
+        b.click("#group-details .pf-c-dropdown button")
+        b.click("#group-details .pf-c-dropdown__menu li:contains('Delete')")
+        b.click("#group-confirm-delete-dialog footer .pf-c-button.apply")
+        b.wait_visible("#groups-list")
+        b.wait_not_in_text('#groups-list', "saturn")
+
+    def testGroupEdit(self):
+        b = self.browser
+        m = self.machine
+
+        m.execute("groupadd titan; useradd uranus")
+
+        self.addCleanup(m.execute, "groupdel titan || true")
+        self.addCleanup(m.execute, "userdel uranus || true")
```

Comment:
This shouldn't be necessary. nonDestructiveSetup() in testlib.py already restores /etc/{passwd,group} and friends wholesale.